openlibrary/templates/subjects.html

```diff
@@ -25,29 +42,13 @@ <h1 class="inline">
 </div>
 
 <div class="contentBody">
-
-  $:render_template("books/custom_carousel", books=page.works, key="subjects", load_more={"url": page.key + ".json", "limit": len(page.works)})
-
-  <div class="head">
-    <h2 class="inline">
-      $_("Publishing History")
-    </h2>
-      <span class="shift">$:_('This is a chart to show the publishing history of editions of works about this subject. Along the X axis is time, and on the y axis is the count of editions published. <a href="#subjectRelated">Click here to skip the chart</a>.')</span>
-      <span class="count hidden chartZoom">&nbsp;<a href="javascript:;" class="resetSelection small">$_("Reset chart")</a> $_("or continue zooming in.")</span>
-      <span class="count chartUnzoom">&nbsp;$_("This graph charts editions published on this subject.")</span>
-  </div>
-
-    <script type="text/json+graph" id="graph-json-chartPubHistory">$:json_encode(page.publishing_history)</script>
-
-    <div class="chart">
-        <div class="chartYaxis">$_("Editions Published")</div>
-        <div id="chartPubHistory" class="thisChart">
-            <noscript>$_("You need to have JavaScript turned on to see the nifty chart!")</noscript>
-        </div>
-        <div class="chartXaxis">$_("Year of Publication")</div>
+  $if has_tag and 'plugins' in page.tag:
+    <div style="padding-bottom: 20px;">
+      $:macros.Plugins(page.tag['plugins'])
     </div>
-
-    <div class="clearfix"></div>
+  $ default_carousel_title = _('Books about %(name)s', name=page.name)
```

Comment:
Let's get feedback from @danafein and @seabelis + the community on the design + language